homeassistant/components/flashforge/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""The FlashForge 3D Printer integration."""
+from __future__ import annotations
+
+from ffpp.Printer import Printer
+
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_IP_ADDRESS, CONF_PORT
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN, SCAN_INTERVAL
+from .data_update_coordinator import FlashForgeDataUpdateCoordinator
+
+PLATFORMS = [SENSOR_DOMAIN]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up FlashForge 3D Printer from a config entry."""
+    printer = Printer(entry.data[CONF_IP_ADDRESS], port=entry.data[CONF_PORT])
+
+    hass.data.setdefault(DOMAIN, {})
+
+    coordinator = FlashForgeDataUpdateCoordinator(hass, printer, entry, SCAN_INTERVAL)
```

Comment:
You can also import the `SCAN_INTERVAL` in your Data Update Coordinator? Is there a specific reason why you would pass it via the constructor.